examples/shader/Program.cs

```diff
@@ -13,10 +13,7 @@ protected Effect(string name)
             myName = name;
         }
 
-        public string Name
-        {
-            get { return myName; }
-        }
+        public string Name => myName;
```

Comment:
it's 4.6.1 for Core 2.0 [Table for lookup](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-platforms-support)